qiskit/providers/ibmq/experiment/experiment.py

```diff
@@ -95,6 +98,9 @@ def update_from_remote_data(self, remote_data: Dict) -> None:
         self._updated_datetime = str_to_utc(remote_data.get('updated_at', None))
         self._uuid = remote_data['uuid']
         self._plot_names = remote_data.get('plot_names', [])
+        self._hub = remote_data.get('hub_id')
+        self._group = remote_data.get('group_id')
+        self._project = remote_data.get('project_id')
```

Comment:
Similar to this, `from_remote_data()` should populate the experiment object with remote data instead of using the credentials in the provider.

Reply:
Done.